frontend/public/src/components/ProgramEnrollmentDrawer.js

```diff
@@ -28,10 +31,58 @@ export class ProgramEnrollmentDrawer extends React.Component<ProgramEnrollmentDr
     }
 
     if (found === undefined) {
-      return <ProgramCourseInfoCard course={course}></ProgramCourseInfoCard>
+      return (
+        <ProgramCourseInfoCard
+          key={course.readable_id}
+          course={course}
+        ></ProgramCourseInfoCard>
+      )
     }
 
-    return null
+    return (
+      <EnrolledItemCard
+        key={found.readable_id}
+        enrollment={found}
+        isProgramCard={true}
+      ></EnrolledItemCard>
+    )
+  }
+
+  isRequired(course) {
+    const { enrollment } = this.props
+
+    console.log(
+      `retval will be ${enrollment.program.requirements.required.indexOf(
+        course.id
+      )}`
+    )
```

Comment:
Were these console.log's (4 of them) for your debugging or intended for production?